src/benchmarks/micro/libraries/System.Net.WebSockets/Perf.WebSocket.cs

```diff
@@ -0,0 +1,52 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+// following benchmarks consume .NET Core 2.1 APIs and are disabled for other frameworks in .csproj file
```

Comment:
not sure this is helpful to reproduce. they're disabled only for .NET Framework anyway.